cli/cmd/dashboard.go

```diff
@@ -23,45 +23,35 @@ var dashboardCmd = &cobra.Command{
 			log.Fatalf("port must be positive, was %d", proxyPort)
 		}
 
-		kubectl, err := k8s.NewKubectl(shell.NewUnixShell())
+		api, err := k8s.NewK8sAPI(shell.NewUnixShell().HomeDir(), kubeconfigPath)
 		if err != nil {
-			log.Fatalf("Failed to start kubectl: %v", err)
+			log.Fatalf("NewK8sAPI() failed with: %+v", err)
```

Comment:
If this is meant to be seen by the user, I'd keep the original message and avoid leaking method names. Even if not part of the UX, I think that this can go stale quickly as we refactor rename/extract methods so I'd avoid it